symengine/lib/symengine_wrapper.pyx

```diff
@@ -2977,11 +2979,16 @@ zoo = c2py(symengine.ComplexInf)
 nan = c2py(symengine.Nan)
 true = c2py(symengine.boolTrue)
 false = c2py(symengine.boolFalse)
+try:
+    import sympy
+    hbar = PyFunction(sympy.hbar, (), sympy.hbar.func, sympy_module)
```

Comment:
@isuruf This seemingly doesn't work as it returns:
```
'module' object has no attribute 'hbar'
```
I tried changing this to:
```
PyFunction(sympy.physics.quantum.constants.hbar, (), sympy.physics.quantum.constants.hbar.func, sympy_module)
```
but even that returns:
```
'module' object has no attribute 'physics'
```